modules/tensor_mechanics/tests/hyperelastic_viscoplastic/one_elem.i

```diff
@@ -0,0 +1,195 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  elem_type = HEX8
+  displacements = 'ux uy uz'
+[]
+
+[Variables]
+  [./ux]
+    block = 0
+  [../]
+  [./uy]
+    block = 0
+  [../]
+  [./uz]
+    block = 0
+  [../]
+[]
+
+[Kernels]
+  [./TensorMechanics]
+    disp_z = uz
+    disp_y = uy
+    disp_x = ux
```

Comment:
Use need to update all of these input files to the new syntax:

displacements =  'ux uy uz'